contrib/ansible/vagrant/openstack_config.yml

```diff
@@ -0,0 +1,12 @@
+os_username: eparis
```

Comment:
OK just wondering. looks real alight :smile:. that password...

As for the file I haven't played with this so if vagrant tried to pick up this config by default the end-user might not understand why. I would say having a `.example` file extension would make it very clear. Then just call out the copy  in the README.